js/location.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import { URL } from "./url";
+import { notImplemented } from "./util";
+
+// See also setLocation() in globals.ts
+
+/* TODO lib.dom.d.ts contains interface Location. How do we say that
```

Comment:
We would copy it into `domTypes.d.ts` and import it here.

Reply:
Done